src/main.rs

```diff
@@ -173,8 +173,10 @@ fn main() -> Result<(), failure::Error> {
                     app.size = size;
 
                     // Based on the size of the terminal, adjust the search limit.
-                    let max_limit = 50;
-                    app.large_search_limit = min((f32::from(size.height) / 1.5) as u32, max_limit);
+                    let max_limit = (app.size.height as u32) - 13;
```

Comment:
The max limit of 50 is imposed on us by Spotify, sending more than this will give as a bad response from the API.

Reply:
So you could do 
```rust
max(app.size.height as u32 - 13, 50);
```

Reply:
thanks! fixed it.